store/src/test/java/org/apache/rocketmq/store/ha/autoswitch/AutoSwitchHATest.java

```diff
@@ -72,9 +73,9 @@ public class AutoSwitchHATest {
     private String store2HaAddress;
 
     private BrokerStatsManager brokerStatsManager = new BrokerStatsManager("simpleTest", true);
-    private String tmpdir = System.getProperty("java.io.tmpdir");
-    private String storePathRootParentDir = (StringUtils.endsWith(tmpdir, File.separator) ? tmpdir : tmpdir + File.separator) + UUID.randomUUID();
-    private String storePathRootDir = storePathRootParentDir + File.separator + "store";
+
```

Comment:
CI is not passed
Tests in error: 
  AutoSwitchHATest.testTruncateEpochLogAndChangeMaster:380->checkMessage:193 » ConditionTimeout

Tests run: 148, Failures: 0, Errors: 1, Skipped: 9

Reply:
I have will fixed it